tests/utest/test_listing_rules.py

```diff
@@ -135,15 +136,15 @@ def test_list_filtered(self, robocop_pre_load, msg_0101, msg_0102_0204, capsys):
 
     def test_list_configurables(self, robocop_pre_load, msg_0101_config, capsys):
         robocop_pre_load.config.list_configurables = robocop.config.translate_pattern('*')
-        init_empty_checker(robocop_pre_load, msg_0101_config)
+        init_empty_checker(robocop_pre_load, msg_0101_config, conf_param=1001)
         with pytest.raises(SystemExit):
             robocop_pre_load.list_checkers()
         out, _ = capsys.readouterr()
         assert out == "All rules have configurable parameter 'severity'. " \
                       "Allowed values are:\n    E / error\n    W / warning\n    I / info\n" \
                       "Rule - 0101 [W]: some-message: Some description (enabled)\n" \
                       "    Available configurable(s) for this rule:\n" \
```

Comment:
What about removing `Available configurable(s) for this rule:` ?

Reply:
I can remove it - I will update PR

Reply:
Then you can also decrease the indentation by 1 "tab"

Reply:
Removed, decreased indentation - but I've left 'Available configurable(s)' for exception that is throwed when you're trying to configure rule with non existing parameter.